daemon/worker.c

```diff
@@ -499,6 +501,9 @@ answer_norec_from_cache(struct worker* worker, struct query_info* qinfo,
 			secure = 1;
 			break;
 		case sec_status_indeterminate:
+			EDNS_OPT_LIST_APPEND_EDE(&edns->opt_list_out,
+				worker->scratchpad, LDNS_EDE_DNSSEC_INDETERMINATE, "");
+			/* fallthrough */
```

Comment:
This will add and EDE to all answers if no trust anchor is configured. Not a good idea! We definitely want no EDE's added to positive answers when not explicitly configured like that.
This should be turned on explicitly by a config option, and/or the queries could perhaps ask for it by sending an EDE option in the query? That last idea should be discussed in IETF b.t.w. Can you make a note for that? Thanks!

```suggestion
			EDNS_OPT_LIST_APPEND_EDE(&edns->opt_list_out,
				worker->scratchpad, LDNS_EDE_DNSSEC_INDETERMINATE, "");
			/* fallthrough */
```

Reply:
removed and noted